packages/dds/tree/src/feature-libraries/editable-tree/utilities.ts

```diff
@@ -20,7 +22,9 @@ export function isPrimitive(schema: TreeSchema): boolean {
 	return (
 		schema.value !== ValueSchema.Nothing &&
 		schema.localFields.size === 0 &&
-		schema.globalFields.size === 0
+		schema.globalFields.size === 0 &&
+		schema.extraGlobalFields === false &&
+		schema.extraLocalFields.kind.identifier === forbidden.identifier
```

Comment:
this fixes a bug where map like nodes were incorrectly detected as primitives. There was even a test checking that we got this wrong  which had to be updated :(